### ISSUE TITLE
Some built-ins installed with incorrect property descriptors

### ISSUE THREAD

**Issue by zloirock** [oven-sh/bun#399]
For example,
```js
console.log(Object.getOwnPropertyDescriptor(globalThis, 'DOMException')); // => { writable: false, enumerable: true, configurable: false }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'URL')); // => { writable: true, enumerable: true, configurable: false }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'URLSearchParams')); // => { writable: false, enumerable: true, configurable: false }
```
\- they should be writable and configurable - and it breaks polyfills when they try to reinstall those properties with fixes.

**Comment by Electroid** [comment]
This is partially fixed, but still an issue as of `0.2.2`.

```js
Object.getOwnPropertyDescriptor(globalThis, 'DOMException')    // writable: false, configurable: false (BROKEN)
Object.getOwnPropertyDescriptor(globalThis, 'URL')             // writable: true, configurable: false (partly OK)
Object.getOwnPropertyDescriptor(globalThis, 'URLSearchParams') // writable: true, configurable: true (OK)
```

**Comment by zloirock** [comment]
```js
console.log(Object.getOwnPropertyDescriptor(globalThis, 'atob'));           // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'btoa'));           // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'setTimeout'));     // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'clearTimeout'));   // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'setInterval'));    // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'clearInterval'));  // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'setImmediate'));   // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'clearImmediate')); // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'queueMicrotask')); // => { configurable: false, enumerable: true }
console.log(Object.getOwnPropertyDescriptor(globalThis, 'self'));           // => { configurable: false, enumerable: false }
```
